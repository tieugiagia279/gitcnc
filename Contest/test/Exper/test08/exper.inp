300
25104 6688
59331 7654
46279 8480
6352 8454
79972 6764
22426 2410
23285 5919
92831 2853
52226 6220
20075 2860
86965 8347
42725 1546
49759 4217
71109 8789
45250 9010
82112 9504
17801 1710
40241 1051
65579 4027
37144 9864
16064 20
66775 5918
92665 1283
78392 6094
27143 5034
74569 65
97522 4937
46425 4786
96165 7029
63240 6173
57734 5560
58652 9080
1681 6123
80632 9159
68988 5896
97405 2699
57827 3347
76523 3033
87838 2249
79271 4621
47707 7344
37585 8378
5730 1496
5686 4372
88054 6694
3600 455
54281 3722
94768 7945
25123 7866
64917 7935
41796 482
94397 4338
61839 8786
41159 2251
21046 8688
76560 2729
16198 8042
41193 7582
88288 122
12929 263
10913 9705
4337 1459
13540 6570
31705 473
94314 3781
22818 5990
2106 587
55697 2322
92495 2282
45485 9722
87347 1879
21661 9871
92714 925
62773 118
54704 9320
80885 2575
58040 6462
72909 6662
96975 8586
81026 6194
73255 9621
29373 4593
833 5417
29194 4342
21996 3688
69618 9254
64608 4504
36292 655
95160 8698
25017 1081
47300 4000
42597 1579
30546 1269
12084 1202
48859 5657
24600 9794
83762 43
4932 7668
21707 589
83088 8251
30362 6762
88984 2311
13511 656
12481 4845
99258 2971
95931 8225
41210 5480
51425 7706
37355 2339
19949 7869
96281 5077
14614 1116
5459 8608
15155 1796
81266 8121
5890 2054
5421 9169
74322 8013
77551 5831
12570 7362
38539 4260
88832 1959
44381 1483
81036 1539
95201 1789
89899 7111
49139 5545
73566 5159
75787 7788
71018 6214
28864 6440
6429 3914
42817 2481
91685 3135
83033 6932
69213 8282
4351 1987
62352 5347
97574 6139
25412 3819
28634 5644
10641 7033
146 7168
94266 2077
89997 8957
28805 2815
46667 379
13842 2151
46003 2174
2436 2110
39074 2268
85375 2839
45850 4481
20431 4112
74357 9983
98849 5443
70014 3834
66198 3396
24829 1458
39835 7183
1151 2398
90117 6149
49653 1176
77579 5239
26245 7632
63256 7027
37607 1555
34335 1422
70006 8978
12328 6630
81117 9958
35952 7466
59515 195
35902 9616
81524 1335
43709 943
20128 7299
1864 3423
85612 6478
67404 1810
2625 2144
26132 7630
53375 5914
59685 9577
41601 1619
32608 3936
57050 9412
91642 1096
45122 2471
94400 6765
11380 1122
9636 8981
15617 1754
56273 4497
35146 336
73392 653
2034 2908
38367 1321
43935 7875
75329 8006
83233 7451
21153 9720
74166 8101
41150 7699
82634 2310
11650 3574
38361 6552
11676 2077
14118 2765
90878 1890
11673 3207
5931 711
59177 1025
42575 3157
49655 1371
65483 2567
75307 429
63156 328
37285 2183
80022 8799
35661 5269
52924 6428
69241 4761
94201 6168
99946 6237
88530 776
61420 1956
95724 1084
16570 1503
76685 5851
93717 1742
84676 4274
56044 3412
89122 4764
84044 4228
24112 6073
75628 808
78804 9368
93607 4294
71896 2347
15226 6985
77388 8988
63427 9852
93781 1030
24543 3258
73934 9774
82113 5736
57748 9298
21477 8167
74867 4685
84888 3208
9592 1356
81769 2296
45186 4093
83860 5100
29759 6861
59899 8727
30928 7011
29547 6573
71429 9961
41239 9111
62877 2952
3649 1505
79854 8966
25452 5366
41523 4408
43076 4502
92944 6639
41133 7008
3279 9956
95211 8535
23100 1116
91715 4091
67937 7988
4445 8200
20306 7445
32612 5788
67377 1066
29946 8609
16460 9761
79626 4589
55534 903
94680 9897
50960 4069
48581 6122
99606 3969
65686 7362
77542 1673
26237 4606
52513 7720
65602 7224
52364 3210
72430 3924
6611 7456
15434 8014
3345 3322
53468 4320
28654 3799
8585 7586
80750 5950
