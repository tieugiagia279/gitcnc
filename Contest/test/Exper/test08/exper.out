4
4
10
11
2
3
5
7
3
2
11
3
4
5
7
12
3
4
3
22
3
5
4
4
4
2
3
4
3
5
2
5
8
8
2
3
2
3
3
3
10
9
3
4
10
3
3
2
10
10
5
3
8
2
6
2
5
4
4
3
18
5
11
3
4
7
3
5
5
15
3
16
3
2
16
4
3
4
11
5
8
3
4
3
4
9
2
3
6
3
4
4
2
5
3
18
2
3
2
12
4
3
3
2
3
6
2
11
3
8
2
4
6
4
4
5
8
3
5
9
4
5
4
2
2
4
8
3
6
3
11
2
5
5
4
6
5
6
5
2
6
2
3
2
18
3
4
4
2
6
3
5
3
3
17
5
3
2
4
9
2
8
2
3
10
5
2
4
5
7
15
3
2
13
3
2
5
5
9
3
5
6
5
11
5
3
9
4
4
6
2
9
3
4
4
2
6
2
12
6
7
9
3
8
4
5
8
3
6
5
4
2
2
9
3
3
3
2
3
4
6
3
2
10
5
4
2
2
3
4
3
7
4
7
3
5
2
10
5
4
8
8
19
3
2
6
4
6
2
4
3
2
4
4
7
5
4
8
7
15
16
7
6
7
3
6
8
11
5
25
5
2
7
8
7
5
8
2
14
12
3
2
13
6
5
3
7
2
2
8
5
2
4
5
10
6
3
3
5
4
